packages/pharos/src/components/button/pharos-button.wc.stories.ts

```diff
@@ -289,3 +289,39 @@ export const Forms: Story = {
     </div>
   `,
 };
+
+export const Selected: Story = {
```

Comment:
I'm not sure have a new story for "selected" makes sense here any more, maybe we could update the variants to have a toggle-able state that sets/unsets a11y-pressed>